src/vbk/test/integration/test_setup.cpp

```diff
@@ -136,8 +15,8 @@ IntegrationTestFixture::IntegrationTestFixture() : TestChain100Setup()
         return util_service_impl.compareForks(left, right);
     });
     When(Method(util_service_mock, CheckPopInputs)).AlwaysReturn(true);
-    When(Method(util_service_mock, EvalScript)).AlwaysDo([&](const CScript& script, std::vector<std::vector<unsigned char>>& stack, ScriptError* serror, VeriBlock::Publications* pub, bool with_checks) -> bool {
-        return ::TestEvalScript(script, stack, serror, pub, with_checks);
+    When(Method(util_service_mock, EvalScript)).AlwaysDo([&](const CScript& script, std::vector<std::vector<unsigned char>>& stack, ScriptError* serror, VeriBlock::Publications* pub, VeriBlock::Context* ctx, VeriBlock::PopTxType* type, bool with_checks) -> bool {
+        return VeriBlock::EvalScriptPopTx(script, stack, serror, pub, ctx, type, with_checks);
```

Comment:
Here the EvalScript method was modified to avoid the absense of the VeriBlockPublications in the test generated pop transaction, because the original method produces an error